test/integration/multikueue/dispatcher_test.go

```diff
@@ -0,0 +1,488 @@
+		})
+
+		ginkgo.By("checking the workload was not created in any worker cluster", func() {
+			managerWl := &kueue.Workload{}
+			gomega.Expect(managerTestCluster.client.Get(managerTestCluster.ctx, wlLookupKey, managerWl)).To(gomega.Succeed())
+			gomega.Expect(managerWl.Status.ClusterName).To(gomega.BeNil())
+			gomega.Eventually(func(g gomega.Gomega) {
+				g.Expect(worker1TestCluster.client.Get(worker1TestCluster.ctx, wlLookupKey, createdWorkload)).To(utiltesting.BeNotFoundError())
+				g.Expect(worker2TestCluster.client.Get(worker2TestCluster.ctx, wlLookupKey, createdWorkload)).To(utiltesting.BeNotFoundError())
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+		})
+
+		ginkgo.By("updating workload nomination by cluster2", func() {
+			gomega.Eventually(func(g gomega.Gomega) {
+				managerWl := &kueue.Workload{}
+				g.Expect(managerTestCluster.client.Get(managerTestCluster.ctx, wlLookupKey, managerWl)).To(gomega.Succeed())
+				managerWl.Status.NominatedClusterNames = []string{workerCluster2.Name}
+				g.Expect(workload.ApplyAdmissionStatus(managerTestCluster.ctx, managerTestCluster.client, managerWl, true, realClock)).To(gomega.Succeed())
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+		})
+
+		ginkgo.By("checking the workload creation in the worker clusters 2", func() {
+			managerWl := &kueue.Workload{}
+			gomega.Expect(managerTestCluster.client.Get(managerTestCluster.ctx, wlLookupKey, managerWl)).To(gomega.Succeed())
+
+			gomega.Eventually(func(g gomega.Gomega) {
+				g.Expect(worker1TestCluster.client.Get(worker1TestCluster.ctx, wlLookupKey, createdWorkload)).To(utiltesting.BeNotFoundError())
+
+				// The workload should be only created in worker2
+				g.Expect(worker2TestCluster.client.Get(worker2TestCluster.ctx, wlLookupKey, createdWorkload)).To(gomega.Succeed())
+				g.Expect(createdWorkload.Spec).To(gomega.BeComparableTo(managerWl.Spec))
+
+				// nominated workers should be updated in the manager
+				g.Expect(managerTestCluster.client.Get(managerTestCluster.ctx, wlLookupKey, managerWl)).To(gomega.Succeed())
+				ginkgo.GinkgoLogr.Info(fmt.Sprintf("Workload status in manager: %s, %v", managerWl.Status.NominatedClusterNames, managerWl.Status.Conditions))
+				g.Expect(managerWl.Status.NominatedClusterNames).To(gomega.ContainElements(workerCluster2.Name))
+				g.Expect(managerWl.Status.ClusterName).To(gomega.BeNil())
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+		})
+
+		ginkgo.By("updating workload nomination by cluster1 and cluster2", func() {
+			managerWl := &kueue.Workload{}
+			gomega.Eventually(func(g gomega.Gomega) {
+				g.Expect(managerTestCluster.client.Get(managerTestCluster.ctx, wlLookupKey, managerWl)).To(gomega.Succeed())
+				managerWl.Status.NominatedClusterNames = []string{workerCluster1.Name, workerCluster2.Name}
+				g.Expect(workload.ApplyAdmissionStatus(managerTestCluster.ctx, managerTestCluster.client, managerWl, true, realClock)).To(gomega.Succeed())
```

Comment:
Instead of calling ApplyAdmissionStatus can we use regular client status patch, but just passing the field manager? It will be then more easy to see for users they should do it. Now, it is very hidden.

Reply:
We tried, but it requires a lot of changes related to eviction. Here's the PR we created during that attempt: https://github.com/kubernetes-sigs/kueue/pull/6162.

Reply:
I didn't mean to use the Patch status function, just using the client in test directly, without any helpers

Reply:
Ah, I see what you mean.